components/collective-page/sections/Contributions.js

```diff
@@ -237,6 +237,9 @@ class SectionContributions extends React.PureComponent {
 
     if (data.loading) {
       return <LoadingPlaceholder height={600} borderRadius={0} />;
+    } else if (!data.Collective) {
+      console.error(`Empty collective data #${collective.id} in Contributions section`);
+      return null;
```

Comment:
why not rendering/returning some error component or message about the same? 😕

Reply:
why couldn't we add an error message about what has happened, for the cause that next that when something similar happens the user will know the status/acknowledgement and dont come across raising an issue or so

Reply:
Yes, by `Adding that the the PR` I meant that I was planning to add it to the pull request :slightly_smiling_face: 

We won't add a more specific error message here tho because any details we may have will be very technical and not meant to be displayed to the user. The only reason for this code to fail is a crash on the API, so it's the responsibility of our error reporting system to notify us about the exact reason.

Reply:
> We won't add a more specific error message here tho because any details we may have will be very technical and not meant to be displayed to the user.

+1